pkg/googlecloud/marshaler.go

```diff
@@ -20,6 +20,10 @@ type Unmarshaler interface {
 // UUIDHeaderKey is the key of the Pub/Sub attribute that carries Waterfall UUID.
 const UUIDHeaderKey = "_watermill_message_uuid"
 
+// GoogleMessageIdHeaderKey is the key of the Pub/Sub attribute that carries Google Cloud Message ID.
+// This ID is assigned by the server when the message is published and is guaranteed to be unique within the topic.
+const GoogleMessageIdHeaderKey = "_watermill_message_google_message_id"
```

Comment:
```suggestion
const GoogleMessageIDHeaderKey = "_watermill_message_google_message_id"
```

Just a tiny cosmetic suggestion to stick to the usual Go naming convention :) (consider UUID above instead od Uuid)